lib/libConfig.ts

```diff
@@ -17,6 +17,12 @@ import { ImportCSSPlugin } from './plugins/ImportCSS.js'
 type OutputOptions = BuildOptions['rollupOptions']['output']
 
 export interface LibraryOptions extends BaseOptions {
+	/**
+	 * Whether to minify the output
+	 * @default false For libraries the code is not minified by default for better debugging of applications
+	 */
```

Comment:
I'd mention that this is disabled by default not because of debugging,. but because it is just not needed for libraries at all. Otherwise it feels like you still should minify the library.

```suggestion
	/**
	 * Whether to minify the output
	 * @default false For libraries the code is not minified by default for better DX. Usually it is not needed: a library will be minified as a part of an app bundling.
	 */
```

Reply:
:+1: